### DISCUSSION TITLE
十月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1519] [通知]
## 🚀 功能

- 直播页与历史记录页支持自动刷新 @Richasy (#1518)
- 支持视频无痕播放 @Richasy (#1517)

## 🐛 Bug 修复

- 修复关注用户/粉丝页面不能正常打开的问题 @Richasy (#1516)
- 修复消息/稍后再看页面不能正常解析数据的问题 @Richasy (#1515)

## 🧰 管理与维护

- 移除ReactiveUI @Richasy (#1514)
- 更新繁体文本 @jay900604 (#1505)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2210.1.0'>十月更新</a>.</em>

**Comment by Delitriuz** [comment]
感觉好像没用上高刷？好像没网页端滑动那么丝滑
